### ISSUE TITLE
Visual Studio 2019 Preview 1.1 and ASP.NET Core 3.0 Not working with Libman 1.0.163

### ISSUE THREAD

**Issue by dumaisf** [aspnet/LibraryManager#409]
I'm using Visual Studio 2019 Preview 1.1 Version 16.0.0
My project is ASP.NET Core 3.0 preview.18572.1
My LibMan version is 1.0.163+g45474d37ed

In Visual Studio, the context Add / **Client-Side Library**, I add some library:
Here is the resulting Libman.json:

```
{
  "version": "1.0",
  "defaultProvider": "cdnjs",
  "libraries": [
    {
      "ProviderId": "cdnjs",
      "DestinationPath": "wwwroot/lib/animate.css/",
      "IsUsingDefaultProvider": true,
      "IsUsingDefaultDestination": false,
      "Name": "animate.css",
      "Version": "3.7.0"
    },
    {
      "ProviderId": "cdnjs",
      "DestinationPath": "wwwroot/lib/kendo-ui-core/",
      "IsUsingDefaultProvider": true,
      "IsUsingDefaultDestination": false,
      "Name": "kendo-ui-core",
      "Version": "2014.1.416"
    },
    {
      "ProviderId": "cdnjs",
      "DestinationPath": "wwwroot/lib/lightbox2/",
      "IsUsingDefaultProvider": true,
      "IsUsingDefaultDestination": false,
      "Name": "lightbox2",
      "Version": "2.10.0"
    },
    {
      "ProviderId": "cdnjs",
      "DestinationPath": "wwwroot/lib/tinymce/",
      "IsUsingDefaultProvider": true,
      "IsUsingDefaultDestination": false,
      "Name": "tinymce",
      "Version": "4.9.1"
    }
  ]
}
```
When I execute **Restore Client-Side Librairies**, I Get the message:
```
Restore operation started...
Restoring libraries for project Windfall
**The library id is undefined**
Restore operation completed with errors
Time Elapsed: 00:00:00.0160558
========== Finished ==========
```
I expected Libman to restore these libraries in my wwwroot/lib folder and none of these libraries gets restored.

**Comment by dumaisf** [comment]
I created a new **ASP.NET Core 2.2** project (_Latest stable version_). In Visual Studio, with the contextual menu **Add / Client-Side Library**, I added the same libraries and Libman restored my libraries without any problem. The problem was not that Visual Studio was a preview version.

The resulting Libman.json is now very different:
```
{
  "version": "1.0",
  "defaultProvider": "cdnjs",
  "libraries": [
    {
      "library": "animate.css@3.7.0",
      "destination": "wwwroot/lib/animate.css/"
    },
    {
      "library": "kendo-ui-core@2014.1.416",
      "destination": "wwwroot/lib/kendo-ui-core/"
    },
    {
      "library": "lightbox2@2.10.0",
      "destination": "wwwroot/lib/lightbox2/"
    },
    {
      "library": "tinymce@4.9.1",
      "destination": "wwwroot/lib/tinymce/"
    }
  ]
}
```

**Comment by drb-germany** [comment]
I had exactly the same problem: added libraries on my existing project (ASP.NET Core 2.2), libman.json ended the way you describe in your first post (with fields ProviderId, DestinationPath, etc.). Creating a new ASP.NET Core 2.2 project and adding the same library ended in the format of your second post. Copying the content to my original project worked.

**Comment by jimmylewis** [comment]
I believe this was fixed with commit `3c1ce94d2cd` and should show up in VS2019 Preview 2.
